docs/zkapps/o1js/ecdsa.mdx

```diff
@@ -22,48 +22,50 @@ Please note that zkApp programmability is not yet available on Mina Mainnet, but
 
-# Why ECDSA?
+## Why ECDSA?
 
-In order to interact with other blockchains or verify data from the outside world, o1js needs to be able to verify signatures.
-ECDSA is a widely used algorithm that is supported by many libraries and tools. It is also used in Ethereum, so it is important to support it in o1js as well. For example, Ethereum transactions are signed using ECDSA over the secp256k1 curve.
-If a zkApp developer wanted to verify an Ethereum transaction and make a statement about it, they would need to be able to verify the signature of the transaction. This is why ECDSA is important for zkApps.
+To interact with other blockchains and verify data from the outside world, o1js needs to be able to verify signatures.
+ECDSA is a widely used algorithm that is supported by many libraries and tools. For example, Ethereum transactions are signed using ECDSA over the secp256k1 curve.
+As a zkApp developer, when you want to verify an Ethereum transaction and make a statement about it, you must be able to verify the signature of the transaction which is why ECDSA is important for zkApps.
 
-# Basic usage
+## Basic usage
 
-The ECDSA gadget is used to verify ECDSA signatures. It takes as input the message, the signature, and the public key of the signer. It outputs a `Bool` indicating whether the signature is valid or not.
+The ECDSA gadget is used to verify ECDSA signatures. The gadget takes as input the message, the signature, and the public key of the signer. It outputs a `Bool` indicating whether the signature is valid.
 
-Before we can verify a signature, we must initiate the gadget with a curve configuration. The most commonly used curve is secp256k1, which is also used in Ethereum. We can initiate the curve as follows:
+Before you can verify a signature, you must initiate the gadget with a curve configuration. 
+
+To initiate the curve:
 
 ```ts
 // create a secp256k1 curve
 class Secp256k1 extends createForeignCurve(Crypto.CurveParams.Secp256k1) {}
 ```
 
-By default, o1js exports a set of already defined curves. We can use the `createForeignCurve` function to create a curve from a `CurveParams` object. The `CurveParams` object contains the parameters of the curve, such as the modulus, the generator and the parameters `a` and `b` of the cruve equation `y^2 = x^3 + ax + b`.
+By default, o1js exports a set of predefined curves. You can use the `createForeignCurve` function to create a curve from a `CurveParams` object. The `CurveParams` object contains the parameters of the curve, such as the modulus, the generator, and the parameters `a` and `b` of the curve equation `y^2 = x^3 + ax + b`.
 
-The namespace `Crypto.CurveParams` exports a few pre-defined curves, such as `Pallas`, `Vesta` and `Secp256k1`.
+The namespace `Crypto.CurveParams` exports predefined curves, such as `Pallas`, `Vesta`, and `Secp256k1`.
```

Comment:
@Trivo25  line 46 says that o1js exports a set of predefined curves (is that automagic buy default? or we need to include this code?

here we specify that it's the namespace that exports them... same? do we need both mentions?

Reply:
It's exported under the `Crypto` namespace and accessible as `Crypto.CurveParams` if thats what you mean